<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shop</class>
 <widget class="QDialog" name="shop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 900 30pt &quot;Segoe UI&quot;;

text-decoration: underline;</string>
       </property>
       <property name="text">
        <string>Магазин</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_bal">
       <property name="styleSheet">
        <string notr="true">font: 900 25pt &quot;Segoe UI&quot;;

</string>
       </property>
       <property name="text">
        <string>Баланс: </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>16</height>
      </size>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Продати</string>
      </attribute>
      <widget class="QLabel" name="sh_carrot">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>81</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="sh_potato">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>120</y>
         <width>81</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="sh_cucumber">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>120</y>
         <width>81</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="sh_onion">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>20</y>
         <width>81</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="sh_tomato">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>20</y>
         <width>81</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="sh_sunflower">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>120</y>
         <width>81</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>160</y>
         <width>109</width>
         <height>36</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Продати</string>
       </property>
      </widget>
      <widget class="QLabel" name="sh_carrot_3">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>117</y>
         <width>109</width>
         <height>37</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 900 italic 9pt &quot;Segoe UI&quot;;
</string>
       </property>
       <property name="text">
        <string>Картопля: 
10шт/150</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>20</y>
         <width>111</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="sh_carrot_4">
          <property name="styleSheet">
           <string notr="true">font: 900 italic 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Цибуля: 
10шт/300</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Продати</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>120</y>
         <width>111</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="sh_carrot_5">
          <property name="styleSheet">
           <string notr="true">font: 900 italic 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Огірок: 
10шт/250</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Продати</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>120</y>
         <width>111</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="sh_carrot_6">
          <property name="styleSheet">
           <string notr="true">font: 900 italic 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Соняшник: 
10шт/500</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Продати</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_4">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>20</y>
         <width>111</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="sh_carrot_7">
          <property name="styleSheet">
           <string notr="true">font: 900 italic 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Помідори: 
10шт/300</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Продати</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>111</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="sh_carrot_2222">
          <property name="styleSheet">
           <string notr="true">font: 900 italic 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Морква: 
10шт/250</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Продати</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="sh_eggs">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>223</y>
         <width>81</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_16">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>263</y>
         <width>109</width>
         <height>36</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Продати</string>
       </property>
      </widget>
      <widget class="QLabel" name="sh_carrot_13">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>220</y>
         <width>109</width>
         <height>37</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 900 italic 9pt &quot;Segoe UI&quot;;
</string>
       </property>
       <property name="text">
        <string>Яйця: 
3шт/300</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Купити</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>343</y>
         <width>411</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Купити поле (доступно: )</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>160</y>
         <width>109</width>
         <height>36</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Купити</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_5">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>20</y>
         <width>111</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="sh_carrot_8">
          <property name="styleSheet">
           <string notr="true">font: 900 italic 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Цибуля: 
5шт/100</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_10">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Купити</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="sh_onion_2">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>20</y>
         <width>81</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="sh_cucumber_2">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>120</y>
         <width>81</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="sh_carrot_9">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>117</y>
         <width>109</width>
         <height>37</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 900 italic 9pt &quot;Segoe UI&quot;;
</string>
       </property>
       <property name="text">
        <string>Картопля: 
10шт/50</string>
       </property>
      </widget>
      <widget class="QLabel" name="sh_sunflower_2">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>120</y>
         <width>81</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_6">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>111</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="sh_carrot_10">
          <property name="styleSheet">
           <string notr="true">font: 900 italic 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Морква: 
5шт/75</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_11">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Купити</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_7">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>120</y>
         <width>111</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QLabel" name="sh_carrot_11">
          <property name="styleSheet">
           <string notr="true">font: 900 italic 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Огірок: 
10шт/100</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_12">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Купити</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="sh_tomato_2">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>20</y>
         <width>81</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="sh_potato_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>120</y>
         <width>81</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_8">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>120</y>
         <width>111</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QLabel" name="sh_carrot_12">
          <property name="styleSheet">
           <string notr="true">font: 900 italic 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Соняшник: 
10шт/200</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_13">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Купити</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="sh_carrot_14">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>81</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_9">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>20</y>
         <width>111</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <widget class="QLabel" name="sh_carrot_15">
          <property name="styleSheet">
           <string notr="true">font: 900 italic 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Помідори: 
5шт/100</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_15">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Купити</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="sh_feed_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>220</y>
         <width>81</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="sh_feed">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>220</y>
         <width>109</width>
         <height>37</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 900 italic 9pt &quot;Segoe UI&quot;;
</string>
       </property>
       <property name="text">
        <string>Корм: 
10шт/300</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_14">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>263</y>
         <width>109</width>
         <height>36</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Купити</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Відправитись додому</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
